.github/workflows/e2e-test-weekly.yml

```diff
@@ -291,7 +321,7 @@ jobs:
           test: ${{ matrix.test }}
           kubernetesVersion: ${{ matrix.kubernetes-version }}
           provider: ${{ matrix.provider }}
-          selfManagedInfra: ${{ matrix.selfManagedInfra == 'true' }}
+          clusterCreation: ${{ matrix.clusterCreation }}
```

Comment:
we should remove the old tf-module test (at the bottom) and add the provider to the matrix.